
---

std_msgs/Bool success
std_msgs/String message
std_msgs/String[] cs_list